kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -28,9 +28,11 @@ def subsample_fourier(x, k):
         The input tensor periodized along the next to last axis to yield a
         tensor of size x.shape[-2] // k along that dimension.
     """
+    complex_check(x)
+
+    y = x.reshape(-1, k, x.shape[-1] // k)
 
-    N = x.shape[-1]
-    res = x.reshape(x.shape[:-1] + (k, N // k)).mean(axis=(-2,))
+    res = y.mean(axis=(-2,))
```

Comment:
Why this change?

Reply:
I wanted this to be more in line with the 2d and torch backends subsample fourier.

Reply:
Ok.